x-pack/plugins/security_solution/server/search_strategy/timeline/factory/events/details/index.ts

```diff
@@ -32,6 +32,15 @@ export const timelineEventsDetails: SecuritySolutionTimelineFactory<TimelineEven
     const inspect = {
       dsl: [inspectStringifyObject(buildTimelineDetailsQuery(indexName, eventId, docValueFields))],
     };
+
+    if (response.isRunning) {
```

Comment:
NOTE: The changes in this file were ported over from this PR till it's merged (https://github.com/elastic/kibana/pull/88642)